server/tests/replicated_loglet.rs

```diff
@@ -175,4 +182,168 @@ mod tests {
         )
         .await
     }
+
+    #[test(tokio::test)]
+    async fn bifrost_append_and_seal_concurrent() -> googletest::Result<()> {
+        const TEST_DURATION: Duration = Duration::from_secs(60);
```

Comment:
Wouldn't that have a very noticeable impact on the duration of our tests? Perhaps we can make this as 10 seconds by default and we seal every second?